sgx-tools/gen-quote.go

```diff
@@ -75,9 +78,20 @@ For example, generate the quote file according to the given local report file:
 			if len(spid) != spidLength*2 {
```

Comment:
Here spidLength should be imported from intelsgx or attestation package.

Reply:
> Here spidLength should be imported from intelsgx package.

Ok, but I think spidLength should be a public constant of attestation package rather than intelsgx package.  spidLength is about attestaion rather than the architecture.